src/JWTAuth.php

```diff
@@ -15,7 +15,6 @@
 use PHPOpenSourceSaver\JWTAuth\Contracts\Providers\Auth;
 use PHPOpenSourceSaver\JWTAuth\Http\Parser\Parser;
 
-/** @deprecated */
```

Comment:
Why was this removed?

I mean I don't even know why it would be deprecated, but unless we know the why/why not, this should probably stay?

Reply:
No, the original repository the deprecated tag was added but as per rule every time we tag something as deprecated we need to point the replacement and when Tymon did it he just did without replacement (I don't know if nowadays there's a replacement).

And while PHP is evolving very fast I removed the deprecated tag to avoid any language update in the future throwing an exception.

And if you take a look at the class itself there's nothing wrong, what I believe he (Tymon) was doing is moving forward to PHP 8. x adding strong types, etc, and maybe refactoring (even doesn't have any sense with making it deprecated).